qubes-bug-report

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
+
+import subprocess as sub
+import shlex
+import argparse
+import time
+import sys
+
+from os.path import expanduser
+
+#the term qube refers to a qubes vm
+
+def is_program_installed_in_qube( qube_name, command ):
+	is_installed = True
+
+	try:
+		shell_cmd = "qvm-run " + shlex.quote( qube_name ) + " --pass-io --no-color-output 'command -v " + command + "' &> /dev/null"
+		sub.check_call( shell_cmd, shell = True )
+
+	except sub.CalledProcessError:
+		is_installed = False
+
+	return is_installed
+
+
+#this function requires virsh 
+#domstate only works for Xen domU (guests)
+def is_qube_running( qube_name ):
+	runs = False
+
+	out = sub.check_output([ "virsh", "-c", "xen:///", "domstate", shlex.quote( qube_name ) ])
+	out = out.decode('utf-8').replace('\n', '')
+
+	if 'running' == out:
+		runs = True
+
+	return runs
+
+
+def get_qube_packages( qube_name ):
+	content = "## Qubes Packages\n\n"
+
+	#a qube can have more than one package manager installed (only one is functional)
+	pkg_cmd = { 'dpkg' : 'dpkg -l qubes-*', 'pacman' : 'pacman -Qs', 'rpm' : 'rpm -qa qubes-*' }
+
+	if is_qube_running( qube_name ):
+
+		for package_manager in pkg_cmd.keys():
+			if is_program_installed_in_qube( qube_name, package_manager ):
+				pkg_list_cmd = pkg_cmd[package_manager]
+
+				try:
+					shell_cmd = "qvm-run " + shlex.quote( qube_name ) + " --pass-io --no-color-output '" +  pkg_list_cmd + "' 2> /dev/null"
+					out = sub.check_output( shell_cmd, shell = True )
```

Comment:
The qvm-run script writes always to the console, to avoid that I used the `/dev/null` workaround, which did not work properly with the normal `check_output`.